packages/flutter_tools/templates/template_manifest.json

```diff
@@ -326,12 +326,16 @@
         "templates/plugin/ios.tmpl/.gitignore",
         "templates/plugin/ios.tmpl/Assets/.gitkeep",
         "templates/plugin/lib/projectName.dart.tmpl",
+        "templates/plugin/lib/projectName_platform_interface.dart.tmpl",
+        "templates/plugin/lib/projectName_method_channel.dart.tmpl",
+        "templates/plugin/LICENSE.tmpl",
```

Comment:
Were we just not creating this file then? Could you add a quick test for this fix as well?

Reply:
Looks like this might have been a mistake in the merge, as this file is now in templates/plugin_shared/LICENSE.

But, there wasn't a unit test for it either, so I added this file (and README.md) and removed this line.